src/main/scala/tilelink/Bundle_ACancel.scala

```diff
@@ -0,0 +1,66 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.tilelink
+
+import chisel3._
+import chisel3.util._
+import freechips.rocketchip.config.Parameters
+import freechips.rocketchip.diplomacy._
+import freechips.rocketchip.util._
+import scala.collection.immutable.ListMap
+
+class TLBundle_ACancel(val params: TLBundleParameters) extends Record
+{
+  val a =  ReadyValidCancel(new TLBundleA(params))
+  val b = Flipped(Decoupled(new TLBundleB(params)))
+  val c =         Decoupled(new TLBundleC(params))
+  val d = Flipped(Decoupled(new TLBundleD(params)))
+  val e =         Decoupled(new TLBundleE(params))
```

Comment:
It seems to me we should really be doing this all or nothing. Why is it still Decoupled on BCDE?

Reply:
I'm fine doing it all-or-nothing, but I just figured that I should avoid changing everything at once.  So far the A-channel is the only critical path where I've needed this, but the other channels could plausibly want this in the future too.
Should I make this change now?  It will ripple through a handful of places.